content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Micro-Segmentation with Istio Authorization
+description: Describe Istio authorization characterics and how it can be used for different use cases.
+publishdate: 2018-07-20
+subtitle:
+attribution: Limin Wang
+weight: 87
+---
+
+[Istio's authorization feature](/docs/concepts/security/#authorization), also known as Istio Role Based Access Control,
+provides access control for services in an Istio mesh. Istio authorization is Istio’s native authorization support.
+It can be used on any platform that Istio supports. It features:
```

Comment:
I'd also remove the "It can be used on any platform that Istio supports.". That's implied isn't it?

Reply:
Done.